clients/include/testing_auxiliary.hpp

```diff
@@ -304,6 +304,94 @@ void testing_aux_matmul_alg_set_attr_bad_arg(const Arguments& arg) {}
 
 void testing_aux_matmul_alg_get_attr_bad_arg(const Arguments& arg) {}
 
+void testing_aux_matmul_alg_matmul_bad_arg(const Arguments& arg)
```

Comment:
"bad_arg" suffix means that you expect some error code should be returned in certain function.
The below codes didn't have that checking.